test/ffi/datasets.jl

```diff
@@ -74,6 +74,47 @@ end
 end
 
 
+@testset "LGBM_DatasetCreateFromFile" begin
+    # Create a sample .csv file with a header row
+    sample_data = """
+    feature1,feature2,feature3,feature4,feature5
+    0.1,0.2,0.3,0.4,0.5
+    0.6,0.7,0.8,0.9,1.0
+    1.1,1.2,1.3,1.4,1.5
+    """
+    sample_file = "sample_data.csv"
+    open(sample_file, "w") do f
+        write(f, sample_data)
+    end
+
+    # Define dataset parameters as strings
+    params = [
+        "two_round=true header=true label_column=0 weight_column=1 ignore_column=2",
+        "two_round=false header=true label_column=1 weight_column=2 ignore_column=3"
+    ]
+
+    expected_num_data = [3, 3]
+    expected_num_feature = [4, 4]
```

Comment:
need to update this to have difference between the params for e.g. with ignoring columns so the created datasets are different in the number of features